parquet/src/main/java/org/apache/iceberg/parquet/ParquetSchemaUtil.java

```diff
@@ -164,4 +165,62 @@ public Boolean primitive(PrimitiveType primitive) {
     }
   }
 
+  public static Type getListElementType(GroupType array) {
```

Comment:
Iceberg doesn't use `get` in method names. There is probably a more specific verb you should use here instead, like `determine` or `find`.